Hazel/src/Platform/OpenGL/OpenGLVertexArray.cpp

```diff
@@ -1,10 +1,11 @@
 #include "hzpch.h"
 #include "OpenGLVertexArray.h"
+#include "OpenGLBuffer.h"
 
 #include <glad/glad.h>
 
-namespace Hazel {
 
```

Comment:
There is an extra open line here.

Reply:
Resolved in https://github.com/TheCherno/Hazel/pull/66/commits/d91579dd7ff16b35eb4a684faa738f5bf1638b3d